core/src/main/scala/cats/hlist/sequence.scala

```diff
@@ -0,0 +1,90 @@
+/*
+ * Adapted from shapeless-contrib scalaz
+ * https://github.com/typelevel/shapeless-contrib/blob/master/scalaz/main/scala/sequence.scala
```

Comment:
Don't link to master as that URL that file might not always be at that path.

How about `https://github.com/typelevel/shapeless-contrib/blob/v0.4/scalaz/main/scala/sequence.scala`